packages/core/types/src/order/common.ts

```diff
@@ -998,6 +1019,11 @@ export interface OrderDTO {
    */
   original_shipping_tax_total: BigNumberValue
 
+  /**
+   * The refundable total of all order items.
+   */
+  item_refundable_total: BigNumberValue
```

Comment:
suggestion: could this be called just `refundable_total`, this is the amount that is currently refundable on the order right?

Reply:
not really, because the total refundable is solely dependent on the payments. This is the "virtual" refundable total if we consider all the items.
Maybe this is not even relevant to return to be honest. I will remove it